include/mxnet/ndarray.h

```diff
@@ -83,7 +83,7 @@ class NDArray {
  public:
   /*! \brief default constructor */
   NDArray()
-    : entry_(nullptr) {
+    : autograd_entry_(nullptr) {
```

Comment:
why dont we initialize `deferredcompute_entry_` in these constructors?

Reply:
It's not required? We just use the `NodeEntry` default constructor.